cdap-common/src/main/resources/bootstrap/distributed.json

```diff
@@ -18,7 +18,7 @@
     {
       "label": "Create pipeline application",
       "type": "CREATE_APPLICATION",
-      "runCondition": "ONCE",
+      "runCondition": "ALWAYS",
```

Comment:
Will this throw any `ConflictException`? If CDAP gets restarted, it will try to create the app with same version which already exists. If CDAP is upgraded, what will happen to the old version of the pipeline app?

Reply:
it won't throw conflict. On restart, it will just overwrite the existing app with the same thing.  On upgrade, it will update the app to use the newer artifact.